src/schema.ts

```diff
@@ -314,6 +314,46 @@ export const applicationRelations = relations(applications, ({ many }) => ({
   accessTokens: many(accessTokens),
 }));
 
+export const accessGrants = pgTable(
+  "access_grants",
+  {
+    id: uuid("id").$type<Uuid>().primaryKey(),
+    token: text("token").notNull().unique(),
```

Comment:
Honestly debating whether this should've been called `token` or `code` but it really doesn't matter a whole lot.

Reply:
Renamed to `code`